regression/basic_flow.yaml

```diff
@@ -0,0 +1,32 @@
+name: streaming-regression-test-basic-flow
+compute:
+  cluster: r1z1
+gpu_num: 8
+command: |-
+  cd streaming
+  composer -n 1 regression/dataset.py --create --compression gz
+  composer -n 1 regression/dataset.py --delete
+  composer -n 1 regression/dataset.py --create --hashes sha1 xxh128
+  composer -n 2 regression/basic_flow.py
+  composer -n 2 regression/basic_flow.py --local
+  composer -n 2 regression/basic_flow.py --download_retry 10
+  composer -n 2 regression/basic_flow.py --download_timeout 120
+  composer -n 2 regression/basic_flow.py --validate_hash sha1
+  composer -n 2 regression/basic_flow.py --validate_hash xxh128
+  composer -n 2 regression/basic_flow.py --keep_zip
+  composer -n 2 regression/basic_flow.py --predownload 1000
+  composer -n 2 regression/basic_flow.py --cache_limit 64mb
+  composer -n 2 regression/basic_flow.py --num_canonical_nodes 16
+  composer -n 2 regression/basic_flow.py --batch_size 1000
+  composer -n 2 regression/basic_flow.py --shuffle --shuffle_algo py1b --shuffle_seed 12 --shuffle_block_size 10000
+  composer -n 1 regression/dataset.py --delete
+
+image: mosaicml/composer:0.15.0
+scheduling:
+  resumable: true
+  priority: medium
+integrations:
+  - integration_type: git_repo
+    git_repo: b-chu/streaming
```

Comment:
missed changing it to `mosaicml/streaming` ?

Reply:
Is there a good workflow for this? It needs to be set as the fork for testing and then after I see it passes, I change it, but there's no way to test that right?

Reply:
Yeah right! Just remember to change it before merging a PR by adding a new commit.